<?xml version="1.0" encoding="UTF-8"?>

<launch>

	<arg name="launch_prefix" default="" doc="gdb -ex run --args"/>
	<arg name="launch_rviz" default="true" />
	<arg name="cloud_topic" default="/velodyne_points" />
	<arg name="parameter_filename" default="param_velodyne_hdl64.lua"/>
	<arg name="parameter_folder_path" default="$(find open3d_slam_ros)/param/"/>
	<arg name="map_saving_folder" default="$(find open3d_slam_ros)/data/maps/"/>
	<arg name="is_read_from_rosbag" default="false"/>
	<arg name="bag_filename" default="kitti00.bag" /> 
	<arg name="bag_folder_path" default="$(find open3d_slam_ros)/data/rosbags/"/>
	<arg name="use_sim_time" default="true"/>
	<arg name="play_delay" default="0.4" />
	<arg name="play_rate" default="1.0" />

	<include file="$(find open3d_slam_ros)/launch/mapping_rosbag.launch">
		<arg name="cloud_topic" value="$(arg cloud_topic)"/>
		<arg name="launch_rviz" value="$(arg launch_rviz)"/>
		<arg name="parameter_filename" value="$(arg parameter_filename)"/>
		<arg name="parameter_folder_path" value="$(arg parameter_folder_path)"/>
		<arg name="num_accumulated_range_data" value="1"/>
		<arg name="use_sim_time" value="$(arg use_sim_time)"/>
		<arg name="is_read_from_rosbag" value="$(arg is_read_from_rosbag)"/>
		<arg name="bag_folder_path" value="$(arg bag_folder_path)"/>
		<arg name="bag_filename" value="$(arg bag_filename)"/>
		<arg name="play_delay" value="$(arg play_delay)"/>
		<arg name="play_rate" value="$(arg play_rate)"/>
		<arg name="launch_prefix" value="$(arg launch_prefix)"/>
	</include>

</launch>